### ISSUE TITLE
The Write a review button is inaccessible from the keyboard

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons#12509]
### Describe the problem and steps to reproduce it:

1. Sign in to AMO dev
2. Go to an add-on detail page
3. Navigate with the tab key on your keyboard until you reach the *Rate your experience* card
4. Select a star by using your keyboard
5. Try to press tab to move focus to the *Write a review* button

### What happened?

No matter how much you press the tab key, *Write a review* is never focused

### What did you expect to happen?

The *Write a review* button should be focused immediately.

### Anything else we should know?

After this is fixed, there might be another bug: https://github.com/mozilla/addons/issues/12343
